tests/Behat/Gherkin/Filter/AbstractFilterTestBase.php

```diff
@@ -7,7 +7,7 @@
 use Behat\Gherkin\Parser;
 use PHPUnit\Framework\TestCase;
 
-abstract class FilterTest extends TestCase
+abstract class AbstractFilterTestBase extends TestCase
```

Comment:
In our own projects we've gone with `xxTestCase` to follow the `TestCase` base class naming of phpunit, but I think `AbstractXXXTestBase` is also fine.

Reply:
I think I like -TestCase better, I'll go for that.